Dockerfiles/cluster-agent/README.md

```diff
@@ -31,3 +31,100 @@ You'll need to download one of the `datadog-cluster-agent*_amd64.deb` package in
 You can then build the image using `docker build -t datadog/cluster-agent:master .`
 
 If you are on macOS, use the --skip-sign option on the omnibus-build.
+
+## Running the DCA with Kubernetes
+
+To run the DCA in Kubernetes, you can simply run `kubectl create -f dca_deploy.yaml` and use the following manifest
+
+```
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: dca
```

Comment:
maybe we should have a more explicit name or at least `dd-dca` or `datadog-dca`?

Reply:
that's reasonable 👍